src/i18n/es.js

```diff
@@ -37,6 +37,7 @@ export default {
     control_panel_url: `${urlControlPanelMain}`,
     copyright_MD: `© ${year} Doppler LLC. Todos los derechos reservados. [Política de Privacidad y Legales](${urlPrivacy}).`,
     empty_data: `Tu dominio no tiene datos para mostrar en esta sección aún.`,
+    general_error: `¡Ouch! Ocurrio un error inesperado, vuelva a intentar nuevamente`,
```

Comment:
Está fuera del scope del PR, pero me parece que hay una redundancia con _vuelva_ y _nuevamente_ ¿puede ser?

```suggestion
    general_error: `¡Ouch! Ocurrio un error inesperado, vuelva a intentar`,
```


```suggestion
    general_error: `¡Ouch! Ocurrio un error inesperado, intente nuevamente`,
```

Reply:
I agree with this

Reply:
podria ser, es el contenido que definieron los chicos de contenido creo no?  tal vez se podria consultar algun chico de contenido

Reply:
And here remember we try to use neutral language, not argentinian. (intente, vuelva, etc)

Reply:
Good catch @cbernat! ahí corregí my _suggest_

Reply:
changed! and thanks for the remember